### PR TITLE
Add feedback widget to navigation

### LINKED ISSUE #1400: Beta feedback widget in navigation
## Description

As part of the beta test, we want to provide users with a quick way to report backs and feedback about their experience directly within the UI.

**Scope:**

1. Add a navigation item that has a hover states and triggers the menu draw to show with the beta widget inside.
2. The widget should be added to all navigation menu items to ensure users can access it anywhere. 
3. Designs should match Figma and include hover states. 
4. Mobile version should also be completed. 

## Figma Links:

Components and interaction states: https://www.figma.com/file/5V8pr7iMwXsT9L3VAZsmUt/Colony-V5?type=design&node-id=11762-437426&mode=design&t=P1wuBbhYhDDlW38d-4

### PR DESCRIPTION
**Pull request by ArmandoGraterol**
I tried to cover the current variations of the navbar, let me know if I missed one!

- The feedback button/widget should be present in every navbar.
- Clicking the feedback link is supposed to not be wired in in this PR.
- If you hover over the navigation button, it should display the hover state per the specs.
- Mobile version should also be included.

FireShot Capture 233 - Account Profile - Colony - localhost

FireShot Capture 234 - Account Profile - Colony - localhost

FireShot Capture 235 - Account Profile - Colony - localhost

FireShot Capture 236 - Account Profile - Colony - localhost

FireShot Capture 237 - wonker - Colony - localhost

FireShot Capture 238 - wonker - Colony - localhost

Resolves #1400

### CODE REVIEW

**Review by arrenv — DISMISSED** [review]
Very nice @ArmandoGraterol, great work!

Works in and out of Colony context, works in all menus, can't fault it!

File: src/components/v5/frame/NavigationSidebar/NavigationSidebar.tsx [hunk 1]
```diff
@@ -176,7 +177,22 @@ const NavigationSidebarContent: FC<NavigationSidebarProps> = ({
...
+                onClick={() => {
+                  if (openItemIndex === 0) {
+                    setOpenItemIndex(undefined);
+                  } else {
+                    setOpenItemIndex(0);
+                  }
+                }}
+              />
+            </div>
+            {logo ?? logo}
```
Comment (bassgeta) [inline src/components/v5/frame/NavigationSidebar/NavigationSidebar.tsx:193]: did you mean to use conditional rendering here?

**Review by bassgeta — CHANGES_REQUESTED** [review]

**Review by melyndav — DISMISSED** [review]
You smashed this @ArmandoGraterol! Thank you so much!

File: src/components/v5/frame/NavigationSidebar/NavigationSidebar.tsx (hunk 1 above)
Comment (ArmandoGraterol) [reply src/components/v5/frame/NavigationSidebar/NavigationSidebar.tsx]: Good catch! I went a bit further and just added the static logo. I've noticed that in every instance where this sidebar is used (which is 2), we always passed the same logo so I don't think we need the extra prop, at least not for now.

The specs also show the logo for every variation of this sidebar.

**Review by bassgeta — APPROVED** [review]

**Comment by ArmandoGraterol** [comment]
@arrenv @melyndav May I have a re-approve/re-review, please esteemed sir and madam?

**Review by melyndav — DISMISSED** [review]
Reapproving, thanks @ArmandoGraterol!

**Review by arrenv — DISMISSED** [review]
Good to go my man @ArmandoGraterol !!

**Review by arrenv — APPROVED** [review]

**Review by bassgeta — APPROVED** [review]

**Pull request merged**